dotcom-rendering/src/web/components/ShareCount.test.tsx

```diff
@@ -15,6 +16,12 @@ describe('ShareCount', () => {
 	const pageId =
 		'/environment/2020/jan/25/court-probe-carrie-symonds-influence-boris-johnson-badger-cull';
 
+	const defaultFormat = {
+		theme: ArticlePillar.Opinion,
+		display: ArticleDisplay.Standard,
+		design: ArticleDesign.Standard,
+	};
```

Comment:
I think this is perhaps better inlined. The name here is implying something special with this format - that it is the default one - which isn't true and could be confusing.

Reply:
Now it's making more sense to me, thanks :)